arrays_strings/unique_chars/unique_chars_solution.ipynb

```diff
@@ -2,21 +2,30 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
```

Comment:
I am using the 4.3.0 version of Jupyter. I don't know why add that metadata. I delete it.